tests/console/sshd.pm

```diff
@@ -46,7 +46,7 @@ sub run {
         record_info("SuSEfirewall2 not available", "bsc#1090178: SuSEfirewall2 service is not available after upgrade from SLES11 SP4 to SLES15");
     }
     else {
-        systemctl 'stop ' . $self->firewall if !is_virtualization_server;
+        systemctl('stop ' . $self->firewall) if (!is_virtualization_server && script_run("which " . $self->firewall) == 0);
```

Comment:
When you are checking if the binary is present then `!is_virtualization_server` is not needed.